deploy/deployment.yml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ktr
+  name: token-refresher
+  namespace: catalog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: token-refresher
+  template:
+    metadata:
+      labels:
+        app: token-refresher
+    spec:
+      containers:
+      - name: refresher
+        env:
+        - name: KTR_SECRETNAME
+          value: "bearer-creds"
+        - name: KTR_SECRETNAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: KTR_SECRETKEY
+          value: "token"
+        - name: KTR_OIDC_TOKENURL
+          value: "https://auth.vshn.net/token"
```

Comment:
I'm not sure if that should made default? IMO it would be ok if the app crashes if it's left undefined. After all, we kind of expect that this value gets customized in an overlay, right? (I really don't know what the intend is)